python/ray/serve/BUILD

```diff
@@ -50,3 +50,27 @@ py_test(
     tags = ["exclusive"],
     deps = [":serve_lib"]
 )
+
+py_test(
+    name = "tutorial_tensorflow",
+    size = "small",
+    srcs = glob(["examples/doc/*.py"]),
+    tags = ["exclusive"],
+    deps = [":serve_lib"]
+)
+
+py_test(
+    name = "tutorial_pytorch",
```

Comment:
Will this name potentially cause conflicts or are they namespaced to `serve`?

Reply:
These are namespaced